test/RelayHubRegistrationsManagement.test.ts

```diff
@@ -28,8 +28,9 @@ contract('RelayHub Relay Management', function ([_, relayOwner, relayManager, re
   }
 
   beforeEach(async function () {
-    penalizer = await Penalizer.new()
-    relayHub = await deployHub(penalizer.address, hubConfig)
+    relayHub = await deployHub(hubConfig)
+    penalizer = await Penalizer.new(relayHub.address)
+    await relayHub.setPenalizer(penalizer.address)
```

Comment:
If possible, I'd create a utility function to deploy both `RelayHub` and `Penalizer`